packages/visual-group/src/encoder/cartesian-encoder.js

```diff
@@ -176,14 +176,13 @@ export default class CartesianEncoder extends VisualEncoder {
                     only if field is categorical and is not explicitily sorted by user */
                     key = !axisType ? `0${idx}0` : `${idx}00`;
                     const currentFieldName = fieldsObj[axisType][key].oneVar();
-                    const sortingOrder = config.sort[currentFieldName];
-                    const isSortingDisabled = config.sort.disabled;
+                    const sortingOrder = config.sort && config.sort[currentFieldName];
 
-                    if (!isSortingDisabled && sortingOrder) {
+                    if (sortingOrder) {
                         if (sortingOrder === ASCENDING) {
-                            domains[axisType][key].sort();
+                            domains[axisType][key].sort((a, b) => a.localeCompare(b));
```

Comment:
why `localeCompare` in Cartesian Encoder?

Reply:
also, think of a better way to write ascending and descending functions instead of writing if else.

You shud also look for reusing them, since I saw that earlier localeCompare was used

Reply:
Reused the sort function from `muze-utils`